lightwood/mixer/lightgbm.py

```diff
@@ -47,10 +47,31 @@ class LightGBM(BaseMixer):
     use_optuna: bool
     supports_proba: bool
 
+    """
+    Gradient boosting mixer, uses LightGBM as backbone.
+    
+    This mixer is a good all-rounder, due to the generally great performance of tree-based ML algorithms for supervised learning tasks with tabular data.
+    If you want more information regarding the techniques that set apart LightGBM from other gradient boosters, please refer to their technical paper: "LightGBM: A Highly Efficient Gradient Boosting Decision Tree" (2017).
+    
+    We can basically think of this mixer as a wrapper to the LightGBM interface. To do so, there are a few things the user may want to be aware about:
```

Comment:
"As this mixer uses the formal implementation of LightGBM, here are some caveats to be aware of:"